.github/workflows/test-deploy-pnpm.yaml

```diff
@@ -0,0 +1,24 @@
+# Workflow intended to be run only on PRs, that attempts to build locally to catch any errors, using pnpm.
+
+name: Test Deployment - PNPM 
+
+on: 
+  pull_request:
+    # Not specifying a type, lets it run on a PR update 
+    
+jobs:
+  test:
+    runs-on: ubuntu-latest 
+    
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - name: Setup PNPM 
+        run: npm install -g pnpm@v7.0.0
+      - name: Install Dependencies 
+        run: pnpm install 
```

Comment:
Again, I believe it should be use the `--frozen-lockfile` flag to behave like `npm ci`